api/include/awa/static.h

```diff
@@ -248,57 +259,78 @@ AwaError AwaStaticClient_SetLogLevel(AwaLogLevel level);
 AwaError AwaStaticClient_SetEndPointName(AwaStaticClient * client, const char * endPointName);
 
 /**
- * @brief Set the address and port to listen on for incoming CoAP packets. This function must
- *        be called before performing AwaStaticClient_Init.
+ * @brief Set the address and port to listen on for incoming CoAP packets. IPv4 and IPv6 addresses are valid.
+ *        It must be the address of a local interface. Only local DNS names are valid.
+ *
+ *        Address "Any" (`0.0.0.0` for IPv4, `::` for IPv6) is valid.
+ *
+ *        This function must be called before calling ::AwaStaticClient_Init.
  *
  * @param[in] client A pointer to a valid Awa Static Client.
  * @param[in] address IP address to listen on.
  * @param[in] port Port number to listen on.
  * @return AwaError_Success on success.
  * @return AwaError_OperationInvalid if the client has already been initialised.
- * @return AwaError_StaticClientInvalid if the client is NULL.
+ * @return AwaError_StaticClientInvalid if @e client is NULL.
  */
 AwaError AwaStaticClient_SetCoAPListenAddressPort(AwaStaticClient * client, const char * address, int port);
 
 /**
- * @brief Set the URI to connect to a trusted Bootstrap server, in the following
- *        format: "coap://[ip]:port". This function can only be called before
- *        performing AwaStaticClient_Init.
+ * @brief Set the network location of a trusted LWM2M Bootstrap server, in the URI
+ *        format: "coap://[address]:[port]". The client will connect to the bootstrap
+ *        server to obtain the LWM2M Server Bootstrap Information.
+ *
+ *        [address] can be an IPv4 or IPv6 address, or a resolvable DNS name.
+ *
+ *        This function must be called before calling ::AwaStaticClient_Init.
```

Comment:
@david-antliff-imgtec "This function must be called" isn't exactly true, as we can call either this function or set a factory bootstrap configuration.